src/foam/nanos/pm/PM.js

```diff
@@ -88,6 +88,19 @@ foam.CLASS({
     PMLogger pmLogger = (PMLogger) x.get(DAOPMLogger.SERVICE_NAME);
     if ( pmLogger != null ) {
       pmLogger.log(this);
+    } else {
+      foam.nanos.app.AppConfig app = (foam.nanos.app.AppConfig) x.get("appConfig");
```

Comment:
Is this block really needed? If someone doesn't want PM's, they could just not install the PMLogger and they shouldn't be bombarded with exceptions because of it. Thoughts?

Reply:
Having it has revealed a few places where we were trying log PMs and it was not working. If we don't want PMs or Logs then we can install the appropriate Null logger.  On the fence but I keep adding this anytime I'm working/debugging.